packages/shared_preferences/shared_preferences_linux/README.md

```diff
@@ -0,0 +1,23 @@
+# shared_preferences_linux
+
+The Linux implementation of [`shared_preferences`][1].
+
+## Usage
+
+### Import the package
+
+This package has been endorsed, meaning that you only need to add `shared_preferences`
+as a dependency in your `pubspec.yaml`. It will be automatically included in your app
+when you depend on `package:shared_preferences`.
+
+This is what the above means to your `pubspec.yaml`:
+
+```yaml
+...
+dependencies:
+  ...
+  shared_preferences: ^0.6.0
```

Comment:
Note this is a lie currently - it requires the second PR before it is true.

Reply:
Let's not publish lies :) The initial version should say that it needs to be included directly, and that it will be endorsed in the future, then the second PR that adds endorsement should change this README to look like this.

If for some reason the second PR is delayed landing, having incorrect instructions on a published package will be confusing for people.